TheBTeam.BLL/Models/Transaction.cs

```diff
@@ -1,33 +1,45 @@
 ﻿using System;
+using System.ComponentModel.DataAnnotations;
 using Microsoft.VisualBasic;
 
-namespace TheBTeam.BLL.Model
+namespace TheBTeam.BLL.Models
 {
```

Comment:
Do i since we're tinkering with the transactions, do i remember correctly mvc really wanted Id's? if im incorrect no worries